frontend/web/src/app/components/customer-list/customer-list.component.ts

```diff
@@ -88,27 +89,27 @@ export class CustomerListComponent implements OnInit, AfterViewInit {
       });
     });
 
-    (<any>this.table).bootstrapTable('destroy');
-    (<any>this.table).bootstrapTable(
+    (this.table as any).bootstrapTable('destroy');
+    (this.table as any).bootstrapTable(
       {
-        "data": data,
-        "columns": columns,
-        "sortStable": true,
-        "sortName": "name"
-      }
+        data: data,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Expected property shorthand in object literal ('{data}'). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3498)